tests/e2e/local/kind/setup_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Check and setup KinD environment
+# shellcheck source=tests/e2e/local/kind/setup_kind_env.sh
+. ./setup_kind_env.sh
```

Comment:
In documentation it is mentioned to run  ./setup_kind_env.sh manually. Probably should remove this command from here